resources/instructions.md

```diff
@@ -0,0 +1,13 @@
+This tool generates custom-voiced audiobook files from an imported ebook file. Please upload an ebook to begin the conversion process. Output files will be downloaded as a .zip archive.
```

Comment:
This should really be part of your README. Your project isn't complex enough to need multiple files of documentation. Just put this info at the top of the README after the initial intro.

Reply:
This markdown file it used for the streamlit app - I will keep it separate from the README.

As suggested, a link to the HuggingFace Space is in the README, as well as simple instructions for the app.